app-admin/pwman3/pwman3-0.10.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_5 python3_6 )
```

Comment:
Syntaxic sugar for your next PR: `python{3_5,3_6}`

Reply:
Thanks! BASH expansion are indeed usefull.